client/src/components/layout/DogShowPage.js

```diff
@@ -14,24 +18,52 @@ const DogShowPage = (props) => {
       }
       const body = await response.json();
       setDog(body.dog);
+      return true;
     } catch (error) {
       console.log(error);
+      return false;
     }
   };
-  useEffect(() => {
+  const getReviews = async () => {
+    const response = await Fetcher.get(`/api/v1/reviews/${dogId}`);
+    if (response.ok) {
+      setReviews(response.data.reviews);
+    }
+  };
+  const getPageData = async () => {
     getDog();
+    getReviews();
```

Comment:
these two fetch requests should be combined into one. A fetch request that grabs BOTH the dog and the reviews. The cost of doing both is both poor in terms of time efficiency, and in terms of React page load efficiency.